docs/api/en/core/Object3D.html

```diff
@@ -229,6 +229,8 @@ <h3>[method:this copy]( [param:Object3D object], [param:Boolean recursive] )</h3
 		recursive -- if true, descendants of the object are also copied. Default is true.<br /><br />
 
 		Copy the given object into this object.
+
+		Note, the callbacks [page:.onAfterRender] and [page:.onBeforeRender] are not copied across.
```

Comment:
To make this a little more general and cover EventDispatcher methods, let's say:

> Note that event listeners and render callbacks ([page:.onAfterRender] and [page:.onBeforeRender]) are not copied.

Reply:
Would this be better?

    Note: event listeners and user-defined callbacks ([page:.onAfterRender] and [page:.onBeforeRender]) are not copied.

Reply:
sorry for late response, thanks for feedback!